src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -33,14 +24,949 @@ to make sure IE does not go into compatibility mode.
+
+```
+try {
+  aFunctionThatMightFail();
+} catch (err) {
+  Sentry.captureException(err);
+}
+```
+
+{% capture __alert_content -%}
+It's possible to throw strings as errors. In this case, the Sentry SDK will not record tracebacks.
+{%- endcapture -%}
+{%- include components/alert.html
+  title="Note"
+  content=__alert_content
+  level="warning"
+%}
+
+&nbsp;
+### Automatically Capturing Errors
+Sentry attaches global handlers to capture uncaught exceptions and unhandled rejections.
+
+**[Mimi note: Code Snippet]**
+
+&nbsp;
+### Automatically Capturing Errors with Promises
+By default, Sentry for JavaScript captures unhandled promise rejections as described in the official ECMAScript 6 standard.
+
+**[Mimi note: Example Code]**
+
+Configuration may be required if you are using a third-party library to implement promises:
+
+Most promise libraries have a global hook for capturing unhandled errors. You may want to disable default behavior by setting `captureUnhandledRejections` option to `false` and manually hook into such event handler and call `Sentry.captureException` or `Sentry.captureMessage` directly.
+
+For example, the [RSVP.js library](https://github.com/tildeio/rsvp.js/) (used by Ember.js) allows you to bind an event handler to a [global error event](https://github.com/tildeio/rsvp.js#error-handling).
+
+```
+RSVP.on('error', function(reason) {
+  Sentry.captureException(reason);
+});
+```
+
+[Bluebird](http://bluebirdjs.com/docs/getting-started.html) and other promise libraries report unhandled rejections to a global DOM event, `unhandledrejection`. In this case, you don't need to do anything; we've already got you covered with the default `captureUnhandledRejections: true` setting.
+
+Please consult your promise library documentation on how to hook into its global unhandled rejection handler, if it exposes one.
+
+&nbsp;
+## Manually Trigger Errors
+
+**[Mimi note: Best for when an app is in a broken state, but there are no explicit errors being thrown. So, you want to trigger an error, possibly with extra context passed in for debugging purposes.]**
+
+**[Mimi note: Yes, there's duplicate info. Still thinking on this. Trying to cater to the kind of dev who has a set flow. Usually, 1. Initialize, 2. optionally configure, 3. Manually trigger an error (because you wanna take it out for a test drive)]**
+
+&nbsp;
+### Capturing Errors / Exceptions
+
+The most common form of capturing is to capture errors.  What can be captured as an
+error will depend on the platform.  In general if you have something that looks like
+an exception it can be captured.  For some SDKs you can also omit the argument to
+`capture_exception` and it will attempt to capture the current exception.
+
+```
+try {
+  aFunctionThatMightFail();
+} catch (err) {
+  Sentry.captureException(err);
+}
+```
+&nbsp;
+### Generate a Custom Error
+**[Mimi note: I'd like to flesh this out more, but need help.]**
```

Comment:
@kamilogorek Are generating custom errors something we expect people to do? Or should we put it down in advanced usage?

Reply:
It's definitely an advanced use-case. Also `new Error('message')` is built-in, custom would be something like `new MyAppError('something')` and they'd have to create a class that extends built-in `Error` one.
We do handle it nicely though, so it's worth mentioning I guess.

Reply:
See: https://github.com/getsentry/sentry-javascript/blob/master/packages/core/src/error.ts

Reply:
Ok, sounds like this gets pushed to the advanced section, @MimiDumpling